crates/sui-framework/packages/sui-framework/tests/crypto/bls12381_tests.move

```diff
@@ -382,6 +382,38 @@ module sui::bls12381_tests {
         let _ = bls12381::hash_to_g1(&vector[]);
     }
 
+    #[test]
+    fun test_to_from_uncompressed() {
```

Comment:
also let's test here: identity (including regression) and a random group element